test/visibility/import/enablesUnqualified/inheritance-subclassListed2b.good

```diff
@@ -0,0 +1,3 @@
+inheritance-subclassListed2b.chpl:37: In function 'main':
+inheritance-subclassListed2b.chpl:39: error: unresolved call 'owned Child.tertiaryMethod()'
+inheritance-subclassListed2b.chpl:39: note: because no functions named tertiaryMethod found in scope
```

Comment:
Why can't this fail with the same error as the 2a versions of this test?

` inheritance-subclassListed2a.chpl:34: error: Cannot find symbol 'Child' in module 'definesTertiary'`

Reply:
I opened a separate issue for the import list not recognizing that it shouldn't follow a private use when validating.  It's #16982